docker/kuberay-autoscaler/autoscaling_config.py

```diff
@@ -0,0 +1,282 @@
+from contextlib import suppress
+import logging
+import math
+import os
+from typing import Dict, Optional
+
+import json
+
+from ray.autoscaler._private.providers import _get_node_provider
+from ray.autoscaler._private.util import validate_config
+
+logger = logging.getLogger(__name__)
+
+# The name of this Ray cluster. Should coincide with the `metadata.name` field of the
+# Ray CR.
+_RAY_CLUSTER_NAME = os.getenv("RAY_CLUSTER_NAME")
```

Comment:
Can we shift this (and the _RAY_CLUSTER_NAMESPACE below) to a command line argument instead of an environment variable? That would be less error prone, since it can't be overwritten by accident and the control flow of how things get passed through is clearer.

Reply:
Good idea, will do.

Reply:
Kept the envs and passed them as command line arguments to avoid including "default" namespace in the command line args.
The command line arguments can't be overwritten at runtime.